c++/src/ByteRLE.cc

```diff
@@ -354,7 +354,7 @@ namespace orc {
     if (bufferStart == bufferEnd) {
       nextBuffer();
     }
-    return *(bufferStart++);
+    return static_cast<signed char>(*(bufferStart++));
```

Comment:
Unlike the other places, this is `static_cast<signed char>`?

Reply:
How about changing the return type from `signed char` to `int8_t`? But this may introduce a larger change.